ajax/libs/PKI.js/package.json

```diff
@@ -30,5 +30,6 @@
         ]
       }
     ]
-  }
+  },
+  "license": "BSD-3-clause"
```

Comment:
`BSD-3-Clause`
Please use upper case. c -> C
Let it as same with https://github.com/cdnjs/cdnjs/blob/master/tools/license-list.json#L49